internal/rbd/controllerserver.go

```diff
@@ -900,7 +900,7 @@ func (cs *ControllerServer) doSnapshotClone(ctx context.Context, parentVol *rbdV
 		cryptErr := parentVol.copyEncryptionConfig(&cloneRbd.rbdImage)
 		if cryptErr != nil {
 			util.WarningLog(ctx, "failed copy encryption "+
-				"config for %q: %v", cloneRbd.String(), cryptErr)
+				"config for %q: %v", cloneRbd, cryptErr)
```

Comment:
for some cases we pass the pointer and in some places we pass it as value for logging please check or else it will again print the volume structure

Reply:
that is why we do code reviews :)

Reply:
`cloneRbd` is a pointer as can be infered from below
https://github.com/ceph/ceph-csi/blob/0ce6ad11524b3241e68057a7fb6f9aa7951e70ce/internal/rbd/controllerserver.go#L869
https://github.com/ceph/ceph-csi/blob/0ce6ad11524b3241e68057a7fb6f9aa7951e70ce/internal/rbd/snapshot.go#L82